tests/foreman/api/test_variables.py

```diff
@@ -849,31 +1269,16 @@ def test_positive_update_hidden_value_in_variable(self):
 
         1. The variable default value is updated
         2. The 'hidden value' flag set to True
-
-        @caseautomation: notautomated
-
-        @CaseLevel: System
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier3
-    def test_positive_hide_empty_default_value(self):
-        """Hiding the empty default value
-
-        @id: 010253f4-5f33-42b8-a40b-796a373670a6
-
-        @steps:
-
-        1. Create variable with valid type and empty value
-        2. Set 'Hidden Value' to true
-
-        @assert:
-
-        1. The 'hidden value' flag set to True
-        2. The default value is empty after hide
-
```

Comment:
Point 1 is covered in test_positive_hide_variable_default_value
Point 2 is not relevant for API as default_value always available.
So stub was removed.